wolox-training/igc-rails
# Author for articles

### Summary
- Added user field for commentaries
- Added access control for articles (can't edit unless it's your own post)
Note: Specs are missing, assign this back to me if you want the relationship to be tested.

### Trello card
https://trello.com/c/0Mvva2Lb/29-agregarle-creador-a-los-articulos-y-comentarios

### Test plan
1. Create a post while logged out.
2. It should redirect to the sign up page.
3. After signing up or signing in, you should be able to create posts.
4. Login as someone else and create a post.
5. It should be possible to edit your own posts but not the one from the first user.
